### ISSUE TITLE
Color contrast for checkboxes and radio button

### ISSUE THREAD

**Issue by franck-paul** [cehfisher/a11y-style-guide#189]
Hi, nice guide, really.

I have a little remark about checkboxes and radio buttons on this page → http://a11y-style-guide.com/style-guide/section-forms.html : I think they should be more contrasted when unckeched/unselected. The light grey may be difficult to see, on white background.

**Comment by cehfisher** [comment]
Updated both the checkboxes and radio buttons. Please review and let me know if any other changes are needed. Thanks!

**Comment by franck-paul** [comment]
It's better but as WCAG 2.0 is concerned the contrast of the checkboxes/radio buttons (#f4f4f4) on white background, even with a darker border still fail to validate AA level. The according contrast ratio is 1.92:1 (1.1:1 without border). Seems to be a little bit too light, no?

Anyway, the border you put might be sufficient for a majority of people, me the first, thank's.
